src/oop/special_class_method/hard/q6/QuestionText.md

```diff
@@ -0,0 +1,33 @@
+# oop-special_class_method-hard-q6
+
+## Question Text
+
+You are writing a Java program that mimics different Vehicles and their attributes. Write a Java class called **Car**
+that has a `double` variable called **topSpeed** and another `String` variable called **type**.
+
+### Car Class:
+
+Your `Car` class must have the following items:
+
+- Fields :
+    - a `double` attribute named **topSpeed** which is not accessible from other classes
+    - a `String` attribute named **type** which is not accessible from other classes
+
+- Constructors:
+    - A constructor which is accessible from other classes and accepts a`double` and a `String` to initialize the
+      **topSpeed** and **type** attributes.
+
+- Methods:
+    - a `setTopSpeed` method which accepts a `double` value, updates the **topSpeed** attribute with it, and is
+      accessible from other classes
+    - a `getTopSpeed` method which is accessible from other classes and returns the value of the **topSpeed** attribute
+    - a `setType` method which accepts a `String` value, updates the **type** attribute with it, and is accessible
+      from other classes
+    - a `getType` method which is accessible from other classes and returns the value of the **type** attribute
+    - a `toString` method which returns a String of the information about the `Car` class of the form
+      `"This Car has a topSpeed of " + topSpeed " with the type " + type + "."`.
```

Comment:
try to follow the x and y substitutions format instead of giving them something they can copy paste in